Add .travis.yml

Hikari9 opened the pull request:
Propose to use Travis for continuous integeration (nodejs v8+).

Hikari9 committed:
Add .travis.yml

erocrizs commented:
I don't know much about Travis, but a lot of open source software uses it. I'm fine with it.

@Vaan52 please look into Travis as our CI. Pass the PR if you approve.

Vaan52 commented:
I'll look into it soon.

Hikari9 committed:
Add Babel ES6 and mocha dependencies to package.json

Commands 'npm test' and 'npm run fix' should now work without errors with this commit.

Hikari9 committed:
Merge remote-tracking branch 'origin/babel-es6' into travis

Hikari9 commented:
Stacking this PR onto #5 to make sure `npm test` passes Travis CI.

Hikari9 committed:
[babel-es6] Do not ignore package-lock.json

Hikari9 committed:
Merge branch 'babel-es6' into travis

Hikari9 committed:
[travis] Add before_script and script modules
